redis_session.c

```diff
@@ -297,6 +300,8 @@ PS_OPEN_FUNC(redis)
         return SUCCESS;
     }
 
+    redis_pool_free(pool TSRMLS_CC);
```

Comment:
But It seems this two lines should be added when save_path is empty or contains only spaces and commas :)